gapic/ads-templates/%namespace/%name/%version/__init__.py.j2

```diff
@@ -108,3 +108,9 @@ __all__ = (
 )
 {% endif %} {# lazy import #}
 {% endblock %}
+
+
+if not sys.version_info >= (3, 7):
+    from pep562 import Pep562
```

Comment:
This would seem to require adding the following to `gapic/ads-templates/setup.py.j2`:

```python
...
 setuptools.setup(
     ....
     install_requires=(
         'google-api-core >= 1.22.2, < 2.0.0dev',
         'googleapis-common-protos >= 1.5.8',
         'grpcio >= 1.10.0',
         'proto-plus >= 1.15.0',
+        'pep562; python_version < 3.7',
         ...
```

Reply:
I believe you can also enable the 3.6 showcase unit tests:

https://github.com/googleapis/gapic-generator-python/blob/81932a2b71e6ca5f424ddc5c52933ad1d452583a/.github/workflows/tests.yaml#L174-L181